xds/server_test.go

```diff
@@ -470,88 +470,6 @@ func (s) TestServeSuccess(t *testing.T) {
 	}
 }
 
-// TestServeWithStop tests the case where Stop() is called before an LDS update
-// is received. This should cause Serve() to exit before calling Serve() on the
-// underlying grpc.Server.
-func (s) TestServeWithStop(t *testing.T) {
```

Comment:
What is the expected behavior of `Serve` now if the server is stopped before an LDS update is received? Should this test be modified to verify the new behavior instead of completely nixing it?

Reply:
I think it's valid to nix. The issue is that previously the newGRPCServer was blocked on a new LDS resource coming in, and then would exit early if stop() was called. Now, there is no such block, so the server immediately goes serving and there is no behavior to test here. I don't really understand the reason this test existed in the first place.